config/presets/local_host_endpoint_list_preset.json.erb

```diff
@@ -1,11 +1,11 @@
 {
   "title": "Example Service Base URL",
   "id": null,
-  "test_suite_id": "service_base_url_test_kit_suite",
+  "test_suite_id": "service_base_url",
   "inputs": [
     {
       "name": "service_base_url_list_url",
-      "value": "<%= ENV['INFERNO_HOST'] %>/custom/service_base_url_test_kit_suite/examples/testBundleValid.json",
+      "value": "<%= ENV['INFERNO_HOST'] %>/custom/service_base_url/examples/testBundleValid.json",
```

Comment:
These all need to be `Inferno::Application['base_url']` because `INFERNO_HOST` doesn't account for inferno being hosted on a non-root path.

Reply:
Ah, good point.  Went through all references to INFERNO_HOST and made sure it pointed to the right thing.